styleguide/source/assets/js/tabs.js

```diff
@@ -1,57 +1,27 @@
-/*=======Resource Tabs=======*/
-
-$('*[data-resource]').click(function(){
-  // Figure out which tab to display.
-  $tab = findTab($(this));
-  // Show the tab.
-  showTab($tab);
-});
-
-// function showTab() - shows and hides the tab.
-function showTab($tab) {
-  $('.--is-active').removeClass('--is-active');
-  // Show the section.
-  $('.ama__resource-tabs__content section#' + $tab.attr('id')).addClass('--is-active');
-  // Put an active state on the tab.
-  $('.ama__resource-tabs__nav li[data-resource="' + $tab.attr('id') + '"]').addClass('--is-active');
-}
-
-// function findTab() - return object that is the section to be displayed.
-function findTab($obj) {
-  // Get the ID for the section to display.
-  var resourceData =_getSectionID($obj);
-  // Find out which tab the sectionID corresponds to.
-  $('.ama__resource-tabs section').each(function() {
-    var attrID = $(this).attr('id');
-    if(attrID.indexOf(resourceData['sectionID']) >= 0) {
-      $tab = $(this);
-    }
-  });
-  return $tab;
-}
-
-// function _getSectionID() - return array containing the specific resource to show and the section element's ID attribute.
-function _getSectionID($obj) {
-  // Parse out the section ID from the clicked object.
-  var resourceData = [];
-  resourceData['resourceID'] = $obj.attr('data-resource');
-  resourceData['sectionID'] = resourceData['resourceID']
-    .substr(0, resourceData['resourceID'].indexOf('-'));
-  if (!resourceData['sectionID']) {
-    resourceData['sectionID'] = resourceData['resourceID'];
-  }
-  return resourceData;
-}
 /*====== jQuery UI tabs ======*/
 
 
 (function($) {
-    $( ".ama__tabs" ).tabs({
-        activate: function (event, ui) {
-          window.location.hash = ui.newPanel.attr('id');
-          return false;
+    var defaultActiveTab = 0
+    var viewportWidth = window.innerWidth;
+    if (viewportWidth >= 600 && $('.ama__resource-tabs').length > 0) {
+        defaultActiveTab = 1;
+    }
+    $(".ama__tabs").tabs({
+        active: defaultActiveTab,
+        create: function() {
+            var widget = $(this).data('ui-tabs');
+            $(window).on('hashchange', function() {
+                widget.option('active', widget._getIndex(location.hash));
+            });
         }
-      });
+    });
+
+    // Prevent jump onclick
+    $('.ama__resource-tabs__nav a, .ama__tabs-navigation a').on('click', function(e){
```

Comment:
Page seems to still jump. I tested it by making my page a bit shorter and clicking on the tabs